package.json

```diff
@@ -91,7 +91,8 @@
     "test:firefox": "aegir test -t browser -f \"./dist/test/**/*.spec.js\" -- --browser firefox",
     "test:firefox-webworker": "aegir test -t webworker -f \"./dist/test/**/*.spec.js\" -- --browser firefox",
     "test:examples": "cd examples && npm run test:all",
-    "test:interop": "aegir test -t node -f dist/test/interop.js"
+    "test:interop": "aegir test -t node -f dist/test/interop.js",
+    "test:relay": "aegir test -t node -f \"./dist/test/**/{hop,stop,reservation-store,relay,auto-relay}.{node,spec}.js\" --cov"
```

Comment:
Will remove before merging

Reply:
Please can you refrain from adding test code, temporary code, or commented out code etc.  We're all human and these things are too easy to miss.

When ready for review, PRs should be ready to merge and should not need extra pushes after review.

Reply:
Okay, removed